### DISCUSSION TITLE
Able to connect to RD from one computer, but not from other

### DISCUSSION THREAD

**Discussion by bigretromike** [rustdesk/rustdesk#7812] [General]
### Bug Description

Just installed rustdesk on windows 11 home. 1.2.3-2
I'm using 1.2.3-2 on both my computers.
On first computer I'm unable to connect - I'm getting error popup with "Connection Error. Connection time out".
I was thinking its issue on second end, but it looks like I can connect without any issues with that computer from my second computer. Same Windows 11 pro, same spec.
I reboot all 3 of them, and still nothing changed.

I'm running the connection thru vps with hbbr/hbbs 1.1.10-3 on board.
All 3 computers are having same 'Server ID' string.

I had this issues before but usually restarting hbbr/hbbs did the trick, but not this time sadly.

### How to Reproduce

unable to replicate this as I dont know what cause it

### Expected Behavior

able to connect to same machine from 2 different (not in the same time)

### Operating system(s) on local side and remote side

local: w11pro, remote w11h

### RustDesk Version(s) on local side and remote side

1.2.3-2

### Screenshots

<ss>

**Comment by bigretromike** [comment]
@rustdesk any clue why this is an issue. I disabled on both sides Windows Defender Firewall, without any changes, there is also nothing in logs - any way I could try to debug it via extra logs on my side or server ?

**Comment by bigretromike** [comment]
Is there no real way to debug this ? 
Something prevents the connection - same LAN (on my side).

**Comment by bigretromike** [comment]
I checked with AnyDesk, TeamViewer and plan VNC with a repeter setup - no issue there. 

Is it me or when issue is converted to disscusion almost no traffic is in this section ?

**Comment by xlionjuan** [comment]
Add `/r` at the end of the ID.

**Reply by bigretromike** [reply]
What should I expected ? What to look for ? 

it does "Force relay" 

Will try it, thanks for suggestion

**Reply by bigretromike** [reply]
`/r` indeed helped, thank you

**Reply by xlionjuan** [reply]
RustDesk may just not works somehow, and still stuck itself and didn't decide to use relay if direct connection attempts failed 

https://github.com/rustdesk/rustdesk/pull/7900

**Reply by bigretromike** [reply]
oh yes, the infinity loop-of-direct-connect :-) been there, done that 👍 `/r` works like a charm.
